src/omdb_data_merge.sql

```diff
@@ -134,4 +99,116 @@ INSERT INTO media_production_company (media_id, production_company_id, type)
+            WHEN json_data->>'Source' = 'Metacritic'
+                THEN REPLACE(json_data->>'Value', '/100', '')
+            ELSE NULL
+        END AS value,
+        CASE
+            WHEN o.imdbvotes = 'N/A' OR o.imdbvotes = '' THEN 0
+            WHEN json_data->>'Source' = 'Internet Movie Database'
+                THEN REPLACE(o.imdbvotes, ',', '')::INTEGER
+            WHEN jsonb_array_length(o.ratings::jsonb) = 0
+                AND (o.imdbvotes != 'N/A' OR o.imdbvotes != '')
+                THEN REPLACE(o.imdbvotes,',','')::INTEGER
+            ELSE 0
+        END AS imdbvotes
+    FROM original.omdb_data o
+    JOIN media m ON m.imdb_id = o.tconst
+    LEFT JOIN LATERAL jsonb_array_elements(o.ratings::jsonb) AS json_data ON TRUE
+),  
+
+--updating score with the data from JSON and saving ids of updated rows
+--UPDATE happens only if the vote count is higher than the one in IMDB_DB
+imdb_score_update AS (
+    UPDATE score s
+    SET 
+        value = j.value,
+        vote_count = j.imdbvotes
+    FROM json_extract j
+    WHERE j.media_id = s.media_id
+      AND j.source = 'IMDb'
+      AND s.vote_count < j.imdbvotes
+    RETURNING s.media_id, s.source, 'updated' AS status
+)
+
+--insertion of Metacat data and Rotten tomatoes data as well as adding new data if it is not in score yet
+--using imdb_score_update status to check if it was not updated
+INSERT INTO score (source, value, vote_count, at, media_id)
+SELECT j.source, j.value, j.imdbvotes, CURRENT_TIMESTAMP, j.media_id
+FROM json_extract j
+LEFT JOIN imdb_score_update u ON u.media_id = j.media_id AND u.source = j.source
+LEFT JOIN score s ON j.media_id = s.media_id
+-- Insert IMDb ratings which are not in IMDB_DB
+WHERE (j.media_id != s.media_id
+        AND j.source = 'IMDb')
+-- Insert Other ratings
+    OR (u.status IS NULL 
+        AND j.source != 'IMDb');
+
+
+-- ______________///ADDITIONAL THOUGHTS AND COMMENTS\\\_______________
+--********************************************************************
+--____________________________________________________________________
+
+
+-- Updating countty codes to release
+/*
+Maybe we should discuss it before I start
+
+
+WITH omdb_country AS (
+	SELECT tconst, unnest(string_to_array(country, ', ')) as country
+	FROM original.omdb_data o
+	WHERE o.country !='N/A'
+		AND o.country != ''
+		AND o.country IS NOT NULL),
+country_merge AS (
+	SELECT tconst, country
+	FROM omdb_country o
+	JOIN country c ON o.country = c.name),
+other_countries AS (
+    SELECT o.country
+    FROM omdb_country o
+    LEFT JOIN country_merge cm ON o.tconst = cm.tconst AND o.country = cm.country
+    WHERE cm.country IS NULL)
+
+*/
+
+--PROMOTIONAL MEDIA
+
+/*
+***IDEA***
+What if we create a table and have types in there?
+
+related_media_category(categoryname, parentcategory)
+
+We can have then images, websites, videos without parentcat
+and subcategories like poster, actor, director, premiere for images
+and etc... 
+*/
```

Comment:
would generally just move this to the PR description or as a comment (personally preferer description as it is the first thing you see on a PR)

Reply:
Will change it I just added it to comments of PR